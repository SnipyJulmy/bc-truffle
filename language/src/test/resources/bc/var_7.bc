a = 1
b = 2
a
b
define void f() {
  a
  b
  a = 3
  a
  b
  b = 4
  a
  b
}
f()
halt
