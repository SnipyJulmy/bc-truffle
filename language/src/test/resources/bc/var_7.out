1.replaceAll("\n","")
2
1
2
3
2
3
4
